src/components/Balance/Balance.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const BALANCE = 1000000000000000000
```

Comment:
It's not really a balance, but a NEAR nomination
Also better to write 10**18 and a comment that this is denomination of one near in minimal non divisible units (attoNears)

Reply:
right, added